Automattic/wp-calypso
# Site Editor: Unregister site editor feedback plugin

### Changes proposed in this Pull Request

* remove site editor feedback form 

### Screenshots

#### Before 
![image](https://user-images.githubusercontent.com/47489215/160711132-0a65d1c5-ab7f-491e-950d-78765460a2be.png)
#### After 
![image](https://user-images.githubusercontent.com/47489215/160711774-3b419051-1d85-4f9d-947a-4b04b5813be8.png)

### Testing instructions



- Checkout branch
- Sandbox widgets.wp.com
- Sync wpcom-block-editor changes to sandbox PCYsg-l4k-p2
- Navigate to `calypso/apps/wpcom-block-editor` in terminal
- Execute `yarn dev --sync`
- Verify that the feedback plugin is no longer present in the site editor



Related to #61774
